tests/test_cat.py

```diff
@@ -36,6 +36,9 @@ def setUpClass(cls) -> None:
         cls.cdb.config.linking.train = True
         cls.cdb.config.linking.disamb_length_limit = 5
         cls.cdb.config.general.full_unlink = True
+        cls._temp_logs_folder = tempfile.TemporaryDirectory()
```

Comment:
The dir and all contents will be destroyed on interpreter process stop(?) We want the logs to persist outside of the process running.

For linux this could be configurable via an env var, set within MedCATService or CogStack-ModelServe or something.

By default for *nix we want something like:
`~/.local/share/medcat/logs/`

windows:
`C:\Users\%USERNAME%\.cache\medcat\logs\`